_data/sites.json

```diff
@@ -6895,8 +6895,8 @@
     {
         "name": "Slack",
         "url": "https://my.slack.com/account/settings",
-        "difficulty": "easy",
-        "notes": "If you are the Slack team's primary owner you will need to either delete the team or transfer its ownership before deleting your account",
+        "difficulty": "medium",
```

Comment:
```
hard
```
Whenever it involves contacting customer support, it is hard, because we know this might take a while if ever